### ISSUE TITLE
Can not run on non-Android projects

### ISSUE THREAD

**Issue by djj0809** [ryansgot/smc-gradle-plugin#2]
```
Caused by: groovy.lang.MissingPropertyException: Could not get unknown property 'android' for project ':gd-domain' of type org.gradle.api.Project.
	at org.gradle.internal.metaobject.AbstractDynamicObject.getMissingProperty(AbstractDynamicObject.java:83)
	at org.gradle.internal.metaobject.AbstractDynamicObject.getProperty(AbstractDynamicObject.java:61)
	at org.gradle.api.internal.project.DefaultProject_Decorated.getProperty(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.PogoGetPropertySite.getProperty(PogoGetPropertySite.java:50)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:296)
	at com.fsryan.gradle.smc.SmcPlugin.isAndroidProject(SmcPlugin.groovy:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:384)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1022)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:69)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at com.fsryan.gradle.smc.SmcPlugin$_apply_closure2.doCall(SmcPlugin.groovy:41)
```
I thought should use ``` project.metaClass.hasProperty(project,'android') ``` things like that to determine if is an Android project or not

**Comment by ryansgot** [comment]
Wow. @djj0809 I had no idea that someone else was actually using this. I apologize for the slow turnaround. I'd be happy to fix this up for you.

**Comment by djj0809** [comment]
@ryansgot I'm happy to find such a thing in Github, which is beyond my imagination.
Thanks for your efforts, it did save me a lot of time 😄

**Comment by ryansgot** [comment]
Fixed: https://github.com/ryansgot/smc-gradle-plugin/pull/3

**Comment by ryansgot** [comment]
@djj0809 I've published version 0.0.2 to bintray and jcenter. I've also updated the instructions on the README and created another [repository with sample projects](https://github.com/ryansgot/smc-gradle-plugin-example). I've also added the following issue: https://github.com/ryansgot/smc-gradle-plugin/issues/6
